rust/basic_bitcoin/src/basic_bitcoin/src/bitcoin_wallet.rs

```diff
@@ -0,0 +1,259 @@
+//! A demo of a very bare-bones bitcoin "wallet".
+//!
+//! The wallet here showcases how bitcoin addresses can be be computed
+//! and how bitcoin transactions can be signed. It is missing several
+//! pieces that any production-grade wallet would have, including:
+//!
+//! * Support for address types that aren't P2PKH.
+//! * Caching spent UTXOs so that they are not reused in future transactions.
+//! * Option to set the fee.
+use crate::{bitcoin_api, ecdsa_api};
+use bitcoin::util::psbt::serialize::Serialize as _;
```

Comment:
What is ` as _` useful for ?

Reply:
I was wondering the same thing.   :-)

Reply:
This is historical. At some point, this file was importing `Serialize` from bitcoin and `Serialize` from `serde`. The `as _` avoid the naming conflict, but it's no longer needed so I removed it.